osu.Game/Screens/Play/BreakTracker.cs

```diff
@@ -45,8 +45,12 @@ public BreakTracker(double gameplayStartTime = 0, ScoreProcessor scoreProcessor
         protected override void Update()
         {
             base.Update();
+            updateBreakTime();
+        }
 
-            var time = Clock.CurrentTime;
+        private void updateBreakTime()
+        {
+            var time = Clock?.CurrentTime ?? 0;
```

Comment:
I'm not sure this should be doing anything at all if there's no clock available.

(Also a drive-by change with no clear justification in the description.)

Reply:
Yeah fair point